regulations/views/partial_sxs.py

```diff
@@ -89,9 +89,10 @@ def get_context_data(self, **kwargs):
         label_id = context['label_id']
         part = label_id.split('-')[0]
         notice_id = context['notice_id']
+        version = context['version']
```

Comment:
I checked a few different cases and it looks like `version` should always be available. Definitely the code assumes that `notice_id` is always available, since it relies on it by default. I'm going to take your suggestion to fall back to `notice_id` if version is absent and I think that should do it.